internal/broker/schemas.go

```diff
@@ -56,47 +56,70 @@ func (s *SchemaService) Plans(plans PlansConfig, platformRegion string, cp pkg.C
 	}
 	if createSchema, updateSchema, available := s.BuildRuntimeAWSSchemas(platformRegion); available {
-		outputPlans[BuildRuntimeAWSPlanID] = defaultServicePlan(BuildRuntimeAWSPlanID, BuildRuntimeAWSPlanName, plans, createSchema, updateSchema)
+		outputPlans[BuildRuntimeAWSPlanID] = s.defaultServicePlan(BuildRuntimeAWSPlanID, BuildRuntimeAWSPlanName, plans, createSchema, updateSchema)
 	}
 	if createSchema, updateSchema, available := s.BuildRuntimeGcpSchemas(platformRegion); available {
-		outputPlans[BuildRuntimeGCPPlanID] = defaultServicePlan(BuildRuntimeGCPPlanID, BuildRuntimeGCPPlanName, plans, createSchema, updateSchema)
+		outputPlans[BuildRuntimeGCPPlanID] = s.defaultServicePlan(BuildRuntimeGCPPlanID, BuildRuntimeGCPPlanName, plans, createSchema, updateSchema)
 	}
 	if createSchema, updateSchema, available := s.BuildRuntimeAzureSchemas(platformRegion); available {
-		outputPlans[BuildRuntimeAzurePlanID] = defaultServicePlan(BuildRuntimeAzurePlanID, BuildRuntimeAzurePlanName, plans, createSchema, updateSchema)
+		outputPlans[BuildRuntimeAzurePlanID] = s.defaultServicePlan(BuildRuntimeAzurePlanID, BuildRuntimeAzurePlanName, plans, createSchema, updateSchema)
 	}
 	if azureLiteCreateSchema, azureLiteUpdateSchema, available := s.AzureLiteSchemas(platformRegion); available {
-		outputPlans[AzureLitePlanID] = defaultServicePlan(AzureLitePlanID, AzureLitePlanName, plans, azureLiteCreateSchema, azureLiteUpdateSchema)
+		outputPlans[AzureLitePlanID] = s.defaultServicePlan(AzureLitePlanID, AzureLitePlanName, plans, azureLiteCreateSchema, azureLiteUpdateSchema)
 	}
 	if freemiumCreateSchema, freemiumUpdateSchema, available := s.FreeSchemas(cp, platformRegion); available {
-		outputPlans[FreemiumPlanID] = defaultServicePlan(FreemiumPlanID, FreemiumPlanName, plans, freemiumCreateSchema, freemiumUpdateSchema)
+		outputPlans[FreemiumPlanID] = s.defaultServicePlan(FreemiumPlanID, FreemiumPlanName, plans, freemiumCreateSchema, freemiumUpdateSchema)
 	}
 
 	trialCreateSchema := s.TrialSchema(false)
 	trialUpdateSchema := s.TrialSchema(true)
-	outputPlans[TrialPlanID] = defaultServicePlan(TrialPlanID, TrialPlanName, plans, trialCreateSchema, trialUpdateSchema)
+	outputPlans[TrialPlanID] = s.defaultServicePlan(TrialPlanID, TrialPlanName, plans, trialCreateSchema, trialUpdateSchema)
 
 	ownClusterCreateSchema := s.OwnClusterSchema(false)
 	ownClusterUpdateSchema := s.OwnClusterSchema(true)
-	outputPlans[OwnClusterPlanID] = defaultServicePlan(OwnClusterPlanID, OwnClusterPlanName, plans, ownClusterCreateSchema, ownClusterUpdateSchema)
+	outputPlans[OwnClusterPlanID] = s.defaultServicePlan(OwnClusterPlanID, OwnClusterPlanName, plans, ownClusterCreateSchema, ownClusterUpdateSchema)
 
 	return outputPlans
 }
 
+func (s *SchemaService) defaultServicePlan(id, name string, plans PlansConfig, createParams, updateParams *map[string]interface{}) domain.ServicePlan {
```

Comment:
it was a function, but was moved here and changed to a method to have access to the config